index.html

```diff
@@ -0,0 +1,234 @@
+            </a>
+          </li>
+          <li class="nav-link">
+            <a class="nav-link" href="#" rel="noreferrer noopener">English</a>
+          </li>
+          <li class="nav-link">
+            <a class="nav-link" href="#" rel="noreferrer noopener">My Page</a>
+          </li>
+          <li class="nav-link">
+            <a class="nav-link" href="#" rel="noreferrer noopener">Logout</a>
+          </li>
+        </ul>
+      </nav>
+      <ul class="humb mobile-menu mobile-menu-toggle pos-f-t">
+        <li class="hum"><i class="fas fa-bars"></i></li>
+      </ul>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <ul class="menu">
+          <li>
+            <a href="#">
+              <img
+                id="imag-log"
+                src="./src-image/logo-image.png"
+                alt="logo image"
+              />
+            </a>
+          </li>
+          <li>
+            <a
+              class="navbar-brand"
+              id="about"
+              href="about.html"
+              rel="noreferrer noopener"
+              >About</a
+            >
+          </li>
+          <li>
+            <a class="navbar-brand" href="#" rel="noreferrer noopener"
+              >Program</a
+            >
+          </li>
+          <li>
+            <a class="navbar-brand" href="#" rel="noreferrer noopener">Join</a>
+          </li>
+          <li>
+            <a class="navbar-brand" href="#" rel="noreferrer noopener"
+              >Sponsor</a
+            >
+          </li>
+          <li>
+            <a class="navbar-brand" href="#" rel="noreferrer noopener">News</a>
+          </li>
+          <li>
+            <a class="navbar-brand" id="comp" href="#" rel="noreferrer noopener"
+              >CC Campaign</a
+            >
+          </li>
+        </ul>
+      </nav>
+      <div class="menu-overlay hide-menu">
+        <ul class="mobile-menu-toggle menu-close">
+          <li><i class="fas fa-times" id="cross"></i></li>
+        </ul>
+        <ul class="mobile-menu-items mobile-menu-toggle">
+          <li><a href="about.html" rel="noreferrer noopener">About</a></li>
+          <li><a href="#" rel="noreferrer noopener">Program</a></li>
+          <li><a href="#" rel="noreferrer noopener">Join</a></li>
+          <li><a href="#" rel="noreferrer noopener">Sponsor</a></li>
+          <li><a href="#" rel="noreferrer noopener">News</a></li>
+          <li><a href="#" rel="noreferrer noopener">CC Campaign</a></li>
+        </ul>
+      </div>
+    </header>
+    <section class="home col-12">
+      <video autoplay="" muted="" loop="" id="myVideo">
+        <source
+          src="./vedio-background/Silky_Blue_720P_Motion_Background_Loop.mp4"
+          type="video/mp4"
+        />
+      </video>
+      <h4 class="welcome h4">" Welcome for Online Course "</h4>
+      <p class="det">
+        Activities and events for the new online training courses 2022
+      </p>
```

Comment:
- [ ] Let's make these `h4` and `p` tags to match the design template as the `h4` is too small and the `p` is not bold enough. You can play around with the `font-size` and `font-weight` till you get the style that looks closely to the design template.

- [ ] To improve the semantic structure of your HTML, let's replace the `h4` tag with a `p` tag and the `p` tag with a `h1` tag as this `h1` serves as the page header.

Your Implementation | Cindy's Design
:---:|:---:
![](https://i.imgur.com/GXd803B.png) | ![](https://i.imgur.com/3CcmIVC.png)